helm/amko/templates/clusterrole.yaml

```diff
@@ -13,11 +13,17 @@ rules:
     resources: ["routes"]
     verbs: ["get","watch","list"]
   - apiGroups: [""]
-    resources: ["services", "secrets", "namespaces"]
+    resources: ["pods", "services", "secrets", "namespaces"]
     verbs: ["get", "watch", "list"]
   - apiGroups: ["amko.vmware.com"]
     resources: ["gslbconfigs", "gslbconfigs/status", "globaldeploymentpolicies", "globaldeploymentpolicies/status", "gslbhostrules", "gslbhostrules/status", "amkoclusters", "amkoclusters/status"]
     verbs: ["get","watch","list","patch", "update"]
+  - apiGroups: ["ako.vmware.com"]
+    resources: ["clustersets", "multiclusteringresses"]
+    verbs: ["get","watch","list","patch", "update"]
+  - apiGroups: ["ako.vmware.com"]
+    resources: ["serviceimports"]
+    verbs: ["create","get","watch","list","patch", "update", "delete"]
```

Comment:
There should be a space after the `,` when listing multiple verbs. Failing which yamllint with throw the following error - `too few spaces after comma`.

Reply:
I have resolved the yaml lint errors.